app/templates/frameworks/reuse_declaration.html

```diff
@@ -0,0 +1,73 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}
+  {% block page_title_inner %}
+    Reusing answers from an earlier declaration
+  {% endblock %} – Digital Marketplace
+{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace",
+      },
+      {
+        "link": url_for(".dashboard"),
+        "label": "Your account",
+      },
+      {
+        "link": url_for(".framework_dashboard", framework_slug=current_framework.slug),
+        "label": "Apply to " + current_framework.name,
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+  <div class="grid-row framework-dashboard">
+    <div class="column-two-thirds">
+      {% with
+         heading = self.page_title_inner(),
+         smaller = True
+      %}
+        {% include "toolkit/page-heading.html" %}
+      {% endwith %}
+      <p>In {{ old_framework_application_close_date }}, your organisation completed a declaration for {{ old_framework.name }}.</p>
+      <p>You can reuse some of the answers from that declaration.</p>
+      <br>
+      <p>You’ll need to:</p>
+      <ul class="list-bullet">
+          <li>review the answers you gave before and make sure they’re still correct</li>
+          <li>provide some new answers for this declaration</li>
+      </ul>
+      <br>
+      <br>
+
+      <form method="POST" action="{{ url_for(".reuse_framework_supplier_declaration_post", framework_slug=current_framework.slug, old_framework_slug=old_framework.slug) }}">
+        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+        {%
+          with
+          name = "reuse",
+          question = "Do you want to reuse the answers from your earlier declaration?",
+          inline = true,
+          type = "radio",
+          options = [
+            {"label": "Yes", "value": old_framework.name},
```

Comment:
This is wrong - it should be the slug, not the name that gets passed in.  As it is the API is being hit with a name rather than a slug and throwing a 400 error.